aten/src/ATen/detail/HPUHooksInterface.cpp

```diff
@@ -0,0 +1,25 @@
+#include <ATen/detail/HPUHooksInterface.h>
+#include <c10/util/CallOnce.h>
+
+namespace at {
+namespace detail {
```

Comment:
Please use nested namespace
```suggestion
namespace at::detail {
```

Reply:
not everything is in detail, so I'll leave it as is